analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UseConstantsWhereAppropriate.CSharp11.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public class Program
+    {
+        static readonly IntPtr intPtr1;     // Compliant
+        static readonly UIntPtr uIntPtr1; // Compliant
+        static readonly nint nint1; // Compliant
+        static readonly nuint nuint1; // Compliant
+
+        static readonly IntPtr intPtr2 = 42; // Compliant
+        static readonly IntPtr uIntPtr2 = 42; // Compliant
+        static readonly IntPtr nint2 = 42; // Compliant
+        static readonly IntPtr nuint2 = 42; // Compliant
```

Comment:
Shouldn't these be Noncompliant? They can be constants as well. Additionally I think the type should be different, not always `IntPtr`